docs/src/test-api/class-teststore.md

```diff
@@ -0,0 +1,59 @@
+# class: TestStore
+* since: v1.32
+* langs: js
+
+Playwright Test provides a global `store` object that can be used read/write values to the filesystem. Each value is stored in its own file with the key used as a path relative to `TestConfig.storeDir` (it is `./playwright` by default).
+
+```ts
+import { test, store } from '@playwright/test';
+
+test('get user name', async ({ page, context }) => {
+  await page.goto('/');
+  // Return mock user info from the store.
+  await page.route('**/info/user', route => route.fulfill({ path: store.path('mocks/user.json')}))
+  await page.getByText('My Profile');
+  // Check that the name matches mock data.
+  await expect(page.getByLabel('Name')).toHaveText('John');
+});
+```
+
```

Comment:
Suggestion: `method: TestStore.root` to get the root?

Reply:
Let's wait for now?

Reply:
Added.